src/pudl/analysis/record_linkage/eia_ferc1_record_linkage.py

```diff
@@ -35,25 +35,277 @@
+    ),
+    "capacity_factor": embed_dataframe.ColumnVectorizer(
+        transform_steps=[
+            embed_dataframe.ColumnCleaner(cleaning_function="null_to_zero"),
+            embed_dataframe.NumericSimilarityScorer(
+                method="linear",
+                col1="capacity_factor_ferc1",
+                col2="capacity_factor_eia",
+                output_name="capacity_factor",
+            ),
+        ],
+        columns=["capacity_factor_ferc1", "capacity_factor_eia"],
+    ),
+    "fuel_cost_per_mmbtu": embed_dataframe.ColumnVectorizer(
+        transform_steps=[
+            embed_dataframe.ColumnCleaner(cleaning_function="null_to_zero"),
+            embed_dataframe.NumericSimilarityScorer(
+                method="linear",
+                col1="fuel_cost_per_mmbtu_ferc1",
+                col2="fuel_cost_per_mmbtu_eia",
+                output_name="fuel_cost_per_mmbtu",
+            ),
+        ],
+        columns=["fuel_cost_per_mmbtu_ferc1", "fuel_cost_per_mmbtu_eia"],
+    ),
+    "heat_rate_mmbtu_mwh": embed_dataframe.ColumnVectorizer(
+        transform_steps=[
+            embed_dataframe.ColumnCleaner(cleaning_function="null_to_zero"),
+            embed_dataframe.NumericSimilarityScorer(
+                method="linear",
+                col1="unit_heat_rate_mmbtu_per_mwh_ferc1",
+                col2="unit_heat_rate_mmbtu_per_mwh_eia",
+                output_name="heat_rate_mmbtu_mwh",
+            ),
+        ],
+        columns=[
+            "unit_heat_rate_mmbtu_per_mwh_ferc1",
+            "unit_heat_rate_mmbtu_per_mwh_eia",
+        ],
+    ),
+    "fuel_type_code_pudl": embed_dataframe.ColumnVectorizer(
+        transform_steps=[
+            embed_dataframe.ColumnCleaner(cleaning_function="null_to_empty_str"),
+            embed_dataframe.NumericSimilarityScorer(
+                method="exact",
+                col1="fuel_type_code_pudl_ferc1",
+                col2="fuel_type_code_pudl_eia",
+                output_name="fuel_type_code_pudl",
+            ),
+        ],
+        columns=["fuel_type_code_pudl_ferc1", "fuel_type_code_pudl_eia"],
+    ),
+    "installation_year": embed_dataframe.ColumnVectorizer(
+        transform_steps=[
+            embed_dataframe.NumericSimilarityScorer(
+                method="linear",
+                col1="installation_year_ferc1",
+                col2="installation_year_eia",
+                output_name="installation_year",
+            )
+        ],
+        columns=["installation_year_ferc1", "installation_year_eia"],
+    ),
+}
+
+
+@op
+def get_compiled_input_manager(plants_all_ferc1, fbp_ferc1, plant_parts_eia):
+    """Get :class:`InputManager` object with compiled inputs for model."""
+    inputs = InputManager(plants_all_ferc1, fbp_ferc1, plant_parts_eia)
+    # compile/cache inputs upfront. Hopefully we can catch any errors in inputs early.
+    inputs.execute()
+    return inputs
+
+
+@op
+def get_all_pairs_df(inputs):
```

Comment:
It would be nice to be able to use the same op for `get_all_pairs_df` and `get_train_pairs_df` since they're doing the same thing. Maybe we should pass the inputs to an op that just does the `merge`?

Reply:
I changed this so that `inputs` is passed into one op that returns both `all_pairs_df` and `train_pairs_df`.